src/main/java/org/quantumbadger/redreader/cache/CacheDownload.java

```diff
@@ -130,6 +136,9 @@ private void performDownload(final HTTPBackend.Request request) {
 		request.executeInThisThread(new HTTPBackend.Listener() {
 			@Override
 			public void onError(final RequestFailureType failureType, final Throwable exception, final Integer httpStatus) {
+				if(BaseActivity.getTorStatus()) {
```

Comment:
It's possible to do this only if a reddit request fails by checking:

    initiator.queueType == CacheRequest.DownloadQueueType.REDDIT_API